strings.txt

```diff
@@ -14412,3 +14412,410 @@
+    hu = A navigáció csak a jelenlegi helyzetéből érhető el.
+    de = Die Navigation ist nur von Ihrem aktuellen Standort verfügbar.
+    fi = Navigointi onnistuu vain nykyisestä sijainnistasi.
+    cs = Navigace je dostupná pouze z vašeho současného umístění.
+    it = La navigazione è disponibile solo dalla tua posizione attuale. 
+    nb = Navigering er kun tilgjengelig fra din nåværende beliggenhet. 
+    zh-Hant = 導航只能從您目前的位置開始。
+    zh-Hans = 导航只能从您目前的位置开始。
+    th = สามารถใช้การนำทางได้เฉพาะจากตำแหน่งที่ตั้งปัจจุบันของคุณเท่านั้น
+    ja = ナビゲーションは現在位置からのみ利用できます。
+    ro = Navigația este disponibilă doar având ca punct de pornire locațiacctuală.
+    ar = تتوافر إمكانية التنقل فقط من خلال موقعك الحالي. هل تريد أن نخطط لك طريقاً بديلاً؟
+    sk = Navigácia je dostupná iba z vašej aktuálnej polohy. 
+    es = La navegación solo está disponible desde tu ubicación actual.
+    pl = Nawigacja jest dostępna tylko od twojej bieżącej lokalizacji.
+    nl = Navigatie is uitsluitend beschikbaar vanuit uw huidige locatie.
+    pt = Só é possível navegar a partir da sua localização atual.
+
+  [p2p_reroute_from_current]
+    en = Do you want us to plan a route from your current location?
+    ru = Хотите перестроить маршрут от вашего местоположения?
+    fr = Souhaitez-vous que nous planifiions un itinéraire à partir de votre emplacement actuel ?
+    da = Ønsker du, at vi planlægger en rute fra din nuværende placering?
+    id = Apakah Anda ingin kami merencanakan sebuah rute dari lokasi Anda saat ini?
+    ko = 현재 위치에서 경로를 계획하시겠습니까?
+    sv = Vill du att vi planerar en färdväg från din nuvarande plats?
+    tr = Mevcut konumunuzdan bir rota planlamamızı ister misiniz?
+    uk = Хочете спланувати маршрут із поточного місцезнаходження?
+    vi = Bạn có muốn chúng tôi vạch đường từ vị trí hiện tại của bạn không?
+    hu = Szeretne útvonaltervet készíttetni a jelenlegi pozíciójától?
+    de = Soll eine Route von Ihrer aktuellen Position aus berechnet werden?
+    fi = Haluatko valita vaihtoehtoisen reitin?
+    cs = Chcete, abychom naplánovali trasu z vašeho současného umístění?
+    it = Vuoi che impostiamo il percorso dalla tua posizione corrente?
+    nb = Vil du vi skal planlegge en rute fra din nåværende posisjon?
+    zh-Hant = 你是否想要規劃當前位置的路線？
+    zh-Hans = 你是否想要规划当前位置的路线？
+    th = คุณต้องการให้เราาวงแผนเส้นทางจากสถานที่ตั้งปัจจุบันของคุณหรือไม่?
+    ja = 現在位置からのルートを作成しますか？
+    ro = Doriți să vă planificăm o rută având ca punct de pornire locația actuală?
+    ar = هل ترغب في أن نرسم مسار لك من موقعك الحالي؟
+    sk = Doriți să vă planificăm o rută având ca punct de pornire locația actuală?
+    es = ¿Quieres que planeemos un ruta desde tu ubicación actual?
+    pl = Czy chcesz, byśmy zaplanowali trasę z Twojej bieżącej lokalizacji?
+    nl = Wilt u dat wij een route plannen vanaf uw huidige locatie?
+    pt = Deseja planejar uma rota a partir da sua localização atual?
+
+  [whats_new_where_to_turn]
```

Comment:
There are 2 screens in the "what's new section", each contains a header and the body, thus we have 4 strings.